server/webserver.go

```diff
@@ -253,6 +253,30 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 			return
 		}
 
+		pricePerUnitStr := r.FormValue("pricePerUnit")
```

Comment:
This code diff block looks identical to the next code diff block. Maybe we can pull this into a helper? We could then unit test the helper.

Ex.

```
func setPrice(node *LivepeerNode, r *http.Request) error {
    // Add code here
}
```

Reply:
I didn't want to spent too much time on this since we're looking to overhaul the wizard anyway. But if you want I can do this no problem. 

We can do the same for setting the priceInfo on the node in the handlers

Reply:
Since defining a helper should be pretty straightforward I think its worth moving this request parsing + price info setting code into the helper and just calling the helper